.github/workflows/sdist.yml

```diff
@@ -38,17 +38,9 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Install dependencies
-      run: |
```

Comment:
the idea is to simulate core building and distributing this application. how does this improve the experience here?

Reply:
Artifact building should happen with `build`, as well. `build` will respect PEP 518 requirements, raw `setup.py` commands cannot, which is why the workaround in #39416 was introduced. Making an SDist should be done with `pip install build && python -m build --dist` (or `pipx run build`), not `pip install setuptools wheel numpy && python setup.py sdist --formats=gztar`.

Reply:
can you replace with `pip install build && python -m build --dist` prefer to use the standard tools here

Reply:
pipx is a standard tool, it's part of the PyPA, just like build, setuptools, and pip. I can change this if preferred, but then you are dependent on `setup-python@v2`, while pipx is not.